docs/ISSUE_TEMPLATE.md

```diff
@@ -1,13 +1,20 @@
-Use this template when proposing a new style, component or pattern.
+# New issue template
+
+> Use this template when proposing a new style, component or pattern. 
+
+> If you are suggesting a change to something that already exists in the GOV.UK Design System, please propose it by commenting on the issue for that style, component or pattern. You can find issues for all published content in the ['Published' column](https://github.com/alphagov/govuk-design-system-backlog/projects/1#column-2000330) of the GOV.UK Design System backlog.
```

Comment:
I don't think we can have a link in the template. This template appears in the editor - so it will appear as the original code, not a link

Reply:
OK I'll remove.